server/src/test/java/org/opensearch/cluster/coordination/NoMasterBlockServiceTests.java

```diff
@@ -44,12 +44,12 @@
 
 public class NoMasterBlockServiceTests extends OpenSearchTestCase {
 
-    private NoMasterBlockService noMasterBlockService;
+    private NoMasterBlockService noClusterManagerBlockService;
     private ClusterSettings clusterSettings;
 
     private void createService(Settings settings) {
         clusterSettings = new ClusterSettings(settings, BUILT_IN_CLUSTER_SETTINGS);
-        noMasterBlockService = new NoMasterBlockService(settings, clusterSettings);
+        noClusterManagerBlockService = new NoMasterBlockService(settings, clusterSettings);
```

Comment:
`NoMasterBlockService` -> `NoClusterManagerBlockService`

Reply:
Changed the variable name. 👍